File: app/src/main/java/com/githang/gradledoc/app/others/ContributorsPresenter.java
package com.githang.gradledoc.app.others;

import com.githang.gradledoc.common.Presenter;
import com.githang.gradledoc.common.View;

import java.util.List;

/**
 * @author 黄浩杭 (huanghaohang@parkingwang.com)
 * @version 2016-09-11
 * @since 2016-09-11
 */
class ContributorsPresenter<V extends View<List<Contributor>>> extends Presenter.Base<List<Contributor>, V> {
    ContributorsPresenter(V view) {
        super(view, new ContributorModel());
    }
}

File: app/src/main/java/com/githang/gradledoc/app/others/ContributorsPresenter.kt
package com.githang.gradledoc.app.others

import com.githang.gradledoc.common.Presenter
import com.githang.gradledoc.common.View

/**
 * @author 黄浩杭 (huanghaohang@parkingwang.com)
 * @version 2016-09-11
 * @since 2016-09-11
 */
internal class ContributorsPresenter<V : View<List<Contributor>>>(view: V) : Presenter.Base<List<Contributor>, V>(view, ContributorModel())
